HopsanGUI/OptimizationHandler.cpp

```diff
@@ -1440,13 +1453,13 @@ void OptimizationMessageHandler::objectiveChanged(size_t idx)
 
 void OptimizationMessageHandler::candidatesChanged()
 {
-    mpHandler->plotPoints();
+    mpHandler->plotPoints(true);
```

Comment:
In cases like this I would prefer that the parameter (argument) is of a enum class type, so that you can write a resonable value instead of "true". In this case PointPlotOptions::OnlyCandidates or similar.

Reply:
Fixed!